t/t.go

```diff
@@ -515,8 +519,16 @@ type task struct {
 	isCommon bool
 }
 
+// for custom cluster tests (i.e. those not using default docker-compose.yml)
 func composeFileFor(pkg string) string {
 	dir := strings.Replace(pkg, "github.com/dgraph-io/dgraph/", "", 1)
+	if *arch == "arm64" {
```

Comment:
This is not required if you decide to update the PR based on environment variables,

Reply:
removed